esy.json

```diff
@@ -12,9 +12,8 @@
   },
   "esy": {
     "build": "dune build -p #{self.name}",
-    "buildEnv": { "ODOC_SYNTAX": "re" },
```

Comment:
This is what fixed `esy npm-release`. I don't know why..

Reply:
Weird haha. Good find!